compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/copyPasteAndroidTests/lazy/list/LazyListBeyondBoundsTest.kt

```diff
@@ -296,7 +296,7 @@ class LazyListBeyondBoundsTest {
                 } else {
                     // Assert that the beyond bounds items are present.
                     if (expectedExtraItemsBeforeVisibleBounds()) {
-                        assertThat(placedItems).containsExactly(3, 4, 5, 6, 7)
+                        assertThat(placedItems.sorted()).containsExactly(3, 4, 5, 6, 7)
```

Comment:
From Android's `androidx.kruth.containsExactly` function doc:
> To also test that the contents appear in the given order, make a call to [Ordered.inOrder] on the object returned by this method.

So, [our implementation](https://github.com/JetBrains/compose-multiplatform-core/blob/d9aec78d04bf59f82c5ae82ce0f662e3f9b7b9c9/compose/foundation/foundation/src/skikoTest/kotlin/androidx/compose/foundation/Assert.kt#L68) of this check should be fixed (to not check the order) instead.

PS I also had to port it in one more place in #611

Reply:
Your implementation with Set in https://github.com/JetBrains/compose-multiplatform-core/pull/611 won't work well when there are multiple items, so I decided to use internal-testutils-kmp
@igordmn WDYT? I added todo may be to migrate to internal-testutils fully

Reply:
Does this test work without `placedItems.sorted()` after replacing to `androidx.kruth.assertThat`?

Reply:
Ah, really, forgot to remove ))

Reply:
Yes, it works

Reply:
>WDYT? I added todo may be to migrate to internal-testutils fully

Sounds good